Core.DomainModel/Organization/Organization.cs

```diff
@@ -132,6 +133,8 @@ public Organization()
         public virtual ICollection<DataProtectionAdvisor> DataProtectionAdvisors { get; set; }
 
         public virtual ICollection<UIModuleCustomization> UIModuleCustomizations { get; set; }
+        public virtual ICollection<ItSystemUsage.ItSystemUsage> SuppliedArchives { get; set; }
```

Comment:
what about the old "supplier" property... what happens to that. has migration been tested here?

Reply:
also the property name is odd... it is called SuppliedArchives and is a list of itsystems..
Should be `ArchiveSupplierForItSystems`